tools/template_based_query_generation/src/main/java/KeywordsMapping.java

```diff
@@ -1,122 +1,76 @@
+import com.google.common.collect.ImmutableList;
```

Comment:
Please add some unit test for those newly added classes, or leave some note to add those tests in a future PR

Reply:
Done! Added TODOs for necessary unit tests.